seed/data_importer/tasks.py

```diff
@@ -856,12 +860,20 @@ def _save_pm_meter_usage_data_task(meter_readings, file_pk, progress_key):
             )
             with connection.cursor() as cursor:
                 cursor.execute(sql)
-                key = "{} - {}".format(meter.source_id, meter.get_type_display())
+                key = "{} - {} - {}".format(
```

Comment:
This is slightly different from how the other key in this method is built, so a similar pattern to the [other method](https://github.com/SEED-platform/seed/pull/2145/files#r398051680) cannot be used.